velox/functions/sparksql/String.h

```diff
@@ -40,8 +45,15 @@ struct ChrFunction {
     if (ord < 0) {
       result.resize(0);
     } else {
-      result.resize(1);
-      *result.data() = ord;
+      ord = ord & 0xFF;
+      if (ord < 0x80) {
+        result.resize(1);
+        result.data()[0] = ord;
+      } else {
+        result.resize(2);
+        result.data()[0] = 0xC0 + (ord >> 6);
+        result.data()[1] = 0x80 + (ord & 0x3F);
```

Comment:
> It looks only ASCII string will be returned for this Chr function.

@PHILO-HE  Thanks for the comment!
When the input is between 0x80 and 0xFF, the output will be a utf8 string.